### PR TITLE
[ Keychron K6 ] Migrate folder structure

### PR DESCRIPTION
**Pull request by PhilipMourdjis**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keychron/k6/white/config.h [hunk 1]
```diff
@@ -67,6 +62,19 @@
...
+#define ENABLE_LED_MATRIX_BAND_PINWHEEL
+#define ENABLE_LED_MATRIX_BAND_SPIRAL
+#define ENABLE_LED_MATRIX_CYCLE_LEFT_RIGHT
+#define ENABLE_LED_MATRIX_CYCLE_UP_DOWN
+#define ENABLE_LED_MATRIX_CYCLE_OUT_IN
+#define ENABLE_LED_MATRIX_DUAL_BEACON
+#define ENABLE_LED_MATRIX_WAVE_LEFT_RIGHT
+#define ENABLE_LED_MATRIX_WAVE_UP_DOWN
+
 /* Disable the following animation because they are not interesting in monochrome */
```
Comment (dexter93) [inline keyboards/keychron/k6/white/config.h]: the rgb effects disable list can go away on white. Also for the white LED driver, PWM channels must be enabled explicitely due to RAM issues. See https://github.com/SonixQMK/qmk_firmware/blob/sn32_master/keyboards/keychron/c1/white/config.h#L53-L75

**Comment by PhilipMourdjis** [comment]
I think this is good to merge now, I know that someone else has worked on / tested a via keymap so I don't think there is much point including one here.

**Comment by dexter93** [comment]
looks good so far. nitpicks: 

- that custom keymap is @stdvar 's, not mine
- keymaps should be as standardized in the other restructure PR's to follow stock functionality ( to an extent)

**Comment by PhilipMourdjis** [comment]
> * that custom keymap is @stdvar 's, not mine

Renamed

> * keymaps should be as standardized in the other restructure PR's to follow stock functionality ( to an extent)

Done, with minor additions for all rgb options / nkro enable / RESET on FN2 + Backspace

**Pull request merged**